<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MadMail</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../madico.ico</normaloff>../madico.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 238, 236);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGroupBox" name="group_mail">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>981</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGraphicsView" name="photo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Имя</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>961</width>
       <height>451</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_income">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Входящие</string>
      </attribute>
      <widget class="QTableWidget" name="table_income">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>961</width>
         <height>431</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>От кого</string>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>52</red>
          <green>101</green>
          <blue>164</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тема</string>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>52</red>
          <green>101</green>
          <blue>164</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Текст</string>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>52</red>
          <green>101</green>
          <blue>164</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дата и время</string>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>52</red>
          <green>101</green>
          <blue>164</blue>
         </color>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_sent">
      <attribute name="title">
       <string>Отправленные</string>
      </attribute>
      <widget class="QTableWidget" name="table_sent">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>961</width>
         <height>431</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked</set>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Кому</string>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>52</red>
          <green>101</green>
          <blue>164</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тема</string>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>52</red>
          <green>101</green>
          <blue>164</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Текст</string>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>52</red>
          <green>101</green>
          <blue>164</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дата и время</string>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>52</red>
          <green>101</green>
          <blue>164</blue>
         </color>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="button_new_message">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Pixel Anchor-Jack</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 101, 164);
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>Написать
письмо</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_exit">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>530</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Pixel Anchor-Jack</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);</string>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_update">
     <property name="geometry">
      <rect>
       <x>863</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Pixel Anchor-Jack</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 101, 164);
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_login">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>981</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="textfield_login_password">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>280</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_login_incorrect_pass">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>420</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(204, 0, 0);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Неверный логин или пароль!</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="button_login_registration">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>370</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Pixel Anchor-Jack</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(186, 189, 182);</string>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_login_enter">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>320</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Pixel Anchor-Jack</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(52, 101, 164);</string>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textfield_login_username">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>220</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_login_login">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>200</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_login_passwrod">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>260</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>128</width>
       <height>128</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>textfield_login_password</zorder>
    <zorder>label_login_incorrect_pass</zorder>
    <zorder>button_login_registration</zorder>
    <zorder>button_login_enter</zorder>
    <zorder>label_login_login</zorder>
    <zorder>label_login_passwrod</zorder>
    <zorder>textfield_login_username</zorder>
    <zorder>label_image</zorder>
   </widget>
   <widget class="QGroupBox" name="group_registration">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>981</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPlainTextEdit" name="textfield_reg_notes">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>901</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="textfield_reg_username">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="textfield_reg_password">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="textfield_reg_password_again">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="textfield_reg_name">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="textfield_reg_surname">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>110</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="textfield_reg_father">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>110</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_reg_username">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>*Логин</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_reg_name">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>*Имя</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_reg_surname">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>90</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>*Фамилия</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_reg_father">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>90</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Отчество</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_reg_password">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>*Пароль</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_reg_passwrod_again">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>191</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>*Повторно введите пароль</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_reg_notes">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>381</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Дополнительная информация (адрес, телефон и т.д.)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_reg_back">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>510</y>
       <width>94</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Pixel Anchor-Jack</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);
</string>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_reg_apply">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>500</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Pixel Anchor-Jack</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 101, 164);
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_reg_all_fields">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>510</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Demi</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string>Заполните все *обязательные поля</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_reg_password_doesnt_match">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>198</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string>Пароли не совпадают!</string>
     </property>
    </widget>
   </widget>
   <zorder>group_login</zorder>
   <zorder>group_registration</zorder>
   <zorder>group_mail</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="actionAlpha_0_0_1"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionAlpha_0_0_1">
   <property name="text">
    <string>Версия 0.0.1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
